ddtrace/contrib/starlette/patch.py

```diff
@@ -57,6 +39,17 @@ def patch():
 
     _w("starlette.applications", "Starlette.__init__", traced_init)
 
+    handlers = [
+        "Mount",
+        "Route",
+        "Host",
+        "WebSocketRoute",
```

Comment:
If we are adding this in, we should add tests for it.

Reply:
Removed WebSocketRoute and Host